dockerfiles/docker-compose.yml

```diff
@@ -138,6 +138,10 @@ services:
       - RTD_STRIPE_SECRET
       - RTD_STRIPE_PUBLISHABLE
       - RTD_DJSTRIPE_WEBHOOK_SECRET
+      - RTD_GITHUB_APP_ID
+      - RTD_GITHUB_APP_NAME
+      - RTD_GITHUB_APP_WEBHOOK_SECRET
+      - RTD_GITHUB_APP_PRIVATE_KEY
```

Comment:
Isn't there a way to say `RTD_*` for example, so we don't need to add all these variables here? :)

Reply:
@stsewd ping